README.md

```diff
@@ -130,18 +130,18 @@ you need to create or edit the file `~/.ssh/config`:
 
 ```bash
 Host cumlogin
-    User your_user
+    User your_user_name_provided_by_Florian
     Hostname brome.lab.parisdescartes.fr
     Port 2222
-	PreferredAuthentications publickey
-	IdentityFile ~/.ssh/id_rsa.pub
```

Comment:
there is something wrong here. it should be the public part of your key. you will never send your private key anywhere. I'd prefer to leave this as it is because `id_rsa.pub` is the default name out of `ssh-keygen`.